### ISSUE TITLE
Automated Image Build: Setup Github Actions via Github App integration

### ISSUE THREAD

**Issue by jay-dee7** [containerish/OpenRegistry#191]
Since we're already planning to integrate Github Apps into OpenRegistry, it would be nice to have an option to have container image build and push automated. This can be achieved in many ways but one of the most common ways is to do it via Github Actions.

A large scale of teams already use Github Action for CI/CD and having an option to not moving away would be best in the beginning.

## How to achieve this?
- Let the user authenticate and authorise the OpenRegistry Github App
- Grant Access to a repository
- Scan for a Dockerfile
- If present, create a PR with description comparable to that of a dependabot PR ([example](https://github.com/containerish/OpenRegistry/pull/190)). This PR will contain a new Github Action Workflow that builds and pushes the container image to OpenRegistry. This PR will also instruct the `Repo Owner` to configure secrets for the repo? or maybe we can do it via tokens automatically from the GH APP.
- If not present, let the user know that it doesn't exist and maybe they'd want to create it
- Once the PR is merged, Github would automatically trigger a build and push for the container image.
